### ISSUE TITLE
[Matomo] Track usage of "Shopping @ <shop>" button

### ISSUE THREAD

**Issue by filipefurtad0** [openfoodfoundation/wishlist#269]
## What is the need / problem?

There are user reports pointing out that the "Shopping @ <shop>" button is not clear. To best assess how to improve this we would ideally track the usage of this button, and how often is it used.

![image](https://user-images.githubusercontent.com/49817236/187669679-6f05f2f0-c2f6-4325-8f81-c7629bd613b4.png)

## Which type of users does this problem affect (and how many, if known)?
_- Shoppers_

## Potential solutions that will solve the problem  ?[[brainstorming to list feature candidates]

Track usage of "Shopping @ <shop>" button with Matomo

## Connected wishlist and discovery discussions* [list precedent discussions]

This came up [here](https://openfoodnetwork.slack.com/archives/CG7NJ966B/p1661776367331329?thread_ts=1661774901.651919&cid=CG7NJ966B).

**Comment by filipefurtad0** [comment]
> Track usage of "Shopping @ " button with Matomo

I think the scope of this issue could be to determine:
- on which page the customer is
- is is a logged-in or guest session
 ...when clicking the `"Shopping @ " button`.

I can't think of any other scenario right now, so I'd start off trying to implement with these.

**Comment by RachL** [comment]
@filipefurtad0 yes, a good start is to first understand whether or not the button is used, and perhaps the we can dig: at which frequency etc ?
